### PR TITLE
Add files

### PR DESCRIPTION
**Pull request by mit-mit**

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: lints
+description: Lints that represent the Dart style guide
```
Comment (pq) [inline pubspec.yaml:2]: Maybe we should generalize the mission statement?

> Lints that enforce recommended Dart style

> Official Dart lint rules

Do we want to include flutter as well?

> Official Dart and Flutter lint rules

Dunno...

(Happy to land though and iterate.)

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Files and directories created by pub
+.dart_tool/
+.packages
+
+# Omit commiting pubspec.lock for library packages:
+# https://dart.dev/guides/libraries/private-files#pubspeclock
+pubspec.lock
+
+# Conventional directory for build outputs
+build/
```
Comment (pq) [inline .gitignore:10]: I think we can safely leave this out since there should never be any build output?

File: pubspec.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+version: 0.1.0
+homepage: https://dart.dev/guides/language/effective-dart
```
Comment (pq) [inline pubspec.yaml:4]: Will we be adding documentation of these rulesets to effective dart?  If not, maybe a new targeted page?

> https://dart.dev/guides/language/lints (or linting)

?

**Review by pq — APPROVED** [review]
Some nits.  Happy to land and iterate?

File: pubspec.yaml (hunk 3 above)
Comment (mit-mit) [reply pubspec.yaml:4]: My preference would be that effective Dart does somehow communicate what is scoring and what is recommended lints. Is that an option @munificent ?

File: .gitignore (hunk 2 above)
Comment (mit-mit) [reply .gitignore]: Done

File: pubspec.yaml (hunk 3 above)
Comment (pq) [reply pubspec.yaml:4]: SGTM.  Bob?

File: pubspec.yaml [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+environment:
+  sdk: '>=2.12.0 <3.0.0'
+
+# dependencies:
+# dev_dependencies:
+# NOTE: Code is not allowed in this package. Do not add dependencies.
```
Comment (devoncarew) [inline pubspec.yaml:11]: We should file an issue in the repo to enforce this via CI.

Comment (mit-mit) [reply pubspec.yaml:11]: https://github.com/dart-lang/lints/issues/2

Comment (pq) [reply pubspec.yaml:11]: Happy to take this on. 👍

File: pubspec.yaml (hunk 1 above)
Comment (goderbauer) [reply pubspec.yaml:2]: +1 to keeping this a little more general.

Comment (mit-mit) [reply pubspec.yaml]: Done

File: README.md [hunk 5]
```diff
@@ -0,0 +1 @@
+# Dart lints
```
[the remark this answers is not in the document]
Comment (mit-mit) [reply README.md]: Done

File: pubspec.yaml (hunk 3 above)
Comment (munificent) [reply pubspec.yaml:4]: Mentioning these on "Effective Dart" SGTM.

**Pull request merged**